src/NewProject/add_constraints_form.ui

```diff
@@ -86,6 +85,59 @@
          </property>
         </widget>
        </item>
+       <item>

```

Comment:
do we need this changes?

Reply:
Yes, Ravi said that Software should also have pin assign option in "Add Constraints" window.